ride_confirmation_bp.py

```diff
@@ -0,0 +1,54 @@
+from flask import Blueprint, render_template, request, redirect, url_for, flash, abort
+from flask_login import current_user
+import db_functions
+import email_functions
+
+blueprint = Blueprint('ride_confirmation_bp', __name__)
+
+
+@blueprint.route('/potvrzenijizdy/<chosen>')
+def show_ride_confirmation(chosen):
+	chosen = db_functions.choose_carpool(chosen)
+	user = current_user
+	if user.is_authenticated:
+		return render_template('potvrzeni_jizdy.html', values=chosen)
+	else:
+		flash('Abys mohl nastoupit, musíš se nejdřív přihlásit.', "danger")
+		return render_template('prihlaseni.html')
+
+
+@blueprint.route('/potvrzenijizdy/', methods=['POST'])
+def want_to_board_on():
+	user = current_user
+	chosen = db_functions.choose_carpool(request.form.get("id_jizdy"))
+	id_race = request.form.get("id_zavod")
+	id_ride = request.form.get("id_jizdy")
+
+	if user.is_authenticated:
```

Comment:
I think this `if...else` statement can be removed. It was added almost certainly for debugging purposes only.